node/cmd/guardiand/node.go

```diff
@@ -431,6 +432,7 @@ func init() {
 	promRemoteURL = NodeCmd.Flags().String("promRemoteURL", "", "Prometheus remote write URL (Grafana)")
 
 	chainGovernorEnabled = NodeCmd.Flags().Bool("chainGovernorEnabled", false, "Run the chain governor")
+	coinGeckoApiKey = NodeCmd.Flags().String("coinGeckoApiKey", "", "Coin gecko pro API key. Defaults to off via no API key")
```

Comment:
Nit pick on the string here, wdyt of the suggestion below?

"CoinGecko Pro API key. If no API key is provided, CoinGecko requests may be throttled or blocked"